app/misc.py

```diff
@@ -125,6 +126,14 @@ def is_mod(self, sid, power_level=2):
         """ Returns True if the current user is a mod of 'sub' """
         return is_sub_mod(self.uid, sid, power_level, self.can_admin)
 
+    def is_a_mod(self):
```

Comment:
I know this is currently unused, but I have plans for it later. Can remove from this PR if you want

Reply:
It's alright, but remember that if you plan on calling this function when a user is logged out you should have one with the same name in the `SiteAnon` class that returns always false